opentelekomcloud/config.go

```diff
@@ -377,10 +377,10 @@ func (c *Config) objectStorageV1Client(region string) (*gophercloud.ServiceClien
 	})
 }
 
-func (c *Config) otcV1Client(region string) (*gophercloud.ServiceClient, error) {
-	return openstack.NewOtcV1(c.OsClient, gophercloud.EndpointOpts{
+func (c *Config) otcV1Client(region string) (*golangsdk.ServiceClient, error) {
+	return huaweisdk.NewElbV1(c.HwClient, golangsdk.EndpointOpts{
```

Comment:
I find we also have a NewElasticLoadBalancer in golangsdk, which seems following more conventions and no need to pass a "elb" parameter. how about changing to use that?

Reply:
I see. I will update. Thanks!

Reply:
Done